tests/test_agent.py

```diff
@@ -75,5 +77,13 @@ def test_agent_with_model_server(tmpdir, zipped_moodbot_model):
                       body=f.read(),
                       content_type='application/zip',
                       stream=True)
-    agent = rasa_core.agent.load_from_server(model_server=model_endpoint_config)
+    agent = rasa_core.agent.load_from_server(
+            model_server=model_endpoint_config)
     assert agent.fingerprint == fingerprint
+
+    assert agent.domain.as_dict() == moodbot_domain.as_dict()
+
+    agent_policies = set(utils.module_path_from_instance(p) for p in
```

Comment:
should be 
```
{utils.module_path_from_instance(p) 
for p in agent.policy_ensemble.policies}
```
(so `{}` instead of `set` and different line breaking)